From 748da91637c56b8234cae19e826b192a019f9c1c Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 23 Apr 2015 19:04:10 +0800
Subject: [PATCH 0014/5242] MLK-10724-6 net: fec: add i.mx6ul enet support

commit  3b39d0e9728f9c3ecda0e0254025d0ca3da10d26 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX6UL enet is the lite edition of i.MX6Q enet,
don't support Gbps mode.

Signed-off-by: Fugang Duan <B38611@freescale.com>
(cherry picked from commit: 32a2c3878322d8322a29c52af18de7707134ca1c)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/freescale/fec_main.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/net/ethernet/freescale/fec_main.c b/drivers/net/ethernet/freescale/fec_main.c
index a0574f8..57c85bc 100644
--- a/drivers/net/ethernet/freescale/fec_main.c
+++ b/drivers/net/ethernet/freescale/fec_main.c
@@ -125,6 +125,11 @@
 				FEC_QUIRK_BUG_CAPTURE | FEC_QUIRK_HAS_RACC |
 				FEC_QUIRK_HAS_COALESCE,
 	}, {
+		.name = "imx6ul-fec",
+		.driver_data = FEC_QUIRK_ENET_MAC | FEC_QUIRK_HAS_GBIT |
+				FEC_QUIRK_HAS_BUFDESC_EX | FEC_QUIRK_HAS_CSUM |
+				FEC_QUIRK_HAS_VLAN,
+	}, {
 		/* sentinel */
 	}
 };
-- 
1.7.9.5

